docs/guide/features.md

```diff
@@ -296,6 +296,15 @@ Note that:
 - The glob patterns are treated like import specifiers: they must be either relative (start with `./`) or absolute (start with `/`, resolved relative to project root).
 - The glob matching is done via `fast-glob` - check out its documentation for [supported glob patterns](https://github.com/mrmlnc/fast-glob#pattern-syntax).
 
+You should also be aware that glob imports do not accept variables, You must hard code the imports. One possible solution is as so:
+
+```js
+const location = {
+  posts: Object.entries(import.meta.globEager("/posts/**/*.md")),
+  assets: import.meta.globEager("assets/**/*.{png,jpg}")
+}
+```
```

Comment:
I think the example is a bit hard to read now without more context of what you are trying to do. Let's keep this note simple as another bullet point here that should be enough.

```suggestion
  - You should also be aware that glob imports do not accept variables, you need to directly pass the string pattern.
```